sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/changefeed/implementation/PartitionControllerImpl.java

```diff
@@ -134,9 +133,9 @@ private WorkerTask processPartition(PartitionSupervisor partitionSupervisor, Lea
                         PartitionSplitException ex = (PartitionSplitException) throwable;
                         return self.handleSplit(lease, ex.getLastContinuation());
                     } else if (throwable instanceof TaskCancelledException) {
-                        logger.debug(String.format("Partition %s: processing canceled.", lease.getLeaseToken()));
+                        logger.debug("Partition %s: processing canceled.", lease.getLeaseToken());
```

Comment:
This doesn't log what you intend.
`%s` is for String.format not for slfj4 logger.
slf4j logger doesn't understand `%s` placeholder
please fix.

https://www.slf4j.org/manual.html

Reply:
Fixed.